docs/API_SPEC.md

```diff
@@ -781,3 +781,46 @@ curl -X POST http://localhost:8080/alerts/late-arrivals \
   {"error": "meaningful error message"}
   ```
 
+### 3. Get All Alerts API Endpoint
+
+**Endpoint**: `GET /alerts`
+
+**Description**: Gets all alerts from the database.
+
+**Request Body**:
+
+```bash
+curl -H "Authorization: Bearer <ADMIN_TOKEN>" http://localhost:8080/alerts
+```
+
+**RESPONSES**:
+
+Returns an error of one of these types: `0 - Location mismatch alert`.
+
+- **200 OK**
+
+    ```json
+    [
+      {
+      "ID":"a678f5a6-9731-4741-ad0b-de5efbbffc9b",
+      "Type": 0,
+      "Message": "some message",
+      "UserID":"a678f5a6-9731-4741-ad0b-de5efbbffc9b",
+      "ParkingSpaceID":"a678f5a6-9731-4741-ad0b-de5efbbffc9b"
+      },
+      {"other alerts..."}
+    ]
+  ```
+
+- **400 BAD REQUEST**
```

Comment:
Don't think we return a 400 from this endpoint, afaik it's just 500, check your handler